.eslintrc.js

```diff
@@ -0,0 +1,330 @@
+        "quote-props": [                             // require quotes around object literal property names
+            "error",
+            "consistent"                             // enforces a consistent quote style; in a given object, either all of the properties should be quoted, or none of the properties should be quoted
+        ],
+        "quotes": "off",                             // enforces the consistent use of either backticks, double, or single quotes
+        "radix": [                                   // requires passing the radix parameter when using parseInt()
+            "error",
+            "as-needed"                              // disallows providing the 10 radix
+        ],
+        "require-atomic-updates": "error",           // disallow assignments that can lead to race conditions due to usage of await or yield
+        "require-await": "error",                    // warns async functions which have no await expression
+        "require-unicode-regexp": "off",             // enforce the use of `u` flag on regular expressions
+        "rest-spread-spacing": "error",              // enforce consistent spacing between rest and spread operators and their expressions
+        "semi": "error",                             // requires semicolons at the end of statements
+        "semi-spacing": "error",                     // enforce spacing around a semicolon. Default: {"before": false, "after": true}
+        "semi-style": [                              // enforces that semicolons are at the end of statements
+            "error",
+            "last"
+        ],
+        "sort-imports": "error",                     // requires all import declarations and verifies that all imports are first sorted by the used member syntax and then alphabetically by the first member
+        "sort-keys": "off",                          // requires all property definitions of object expressions and verifies that all variables are sorted alphabetically
+        "sort-vars": "error",                        // requires all variable declaration blocks and verifies that all variables are sorted alphabetically
+        "space-before-blocks": "off",                // enforce consistency of spacing before blocks. It is only applied on blocks that don’t begin on a new line
+        "space-before-function-paren": "off",        // require or disallow a space before function parenthesis
+        "space-in-parens": [                         // enforce consistent spacing directly inside of parentheses
+            "error",
+            "never"                                  // never: enforces zero spaces inside of parentheses
+        ],
+        "space-infix-ops": "off",                    // require spacing around infix operators
+        "space-unary-ops": [                         // enforces consistency regarding the spaces after words unary operators
```

Comment:
not sure if this should be
```suggestion
        "space-unary-ops": [                         // enforces consistency regarding the spaces around unary operators
```
but it's unclear

Reply:
Ah I combined the description of the rule and it's options and formed a weird sentence. I will update it. Your suggestion is correct.